manifests/c/CologneCodeCompany/XYplorer/26.20.0200/CologneCodeCompany.XYplorer.locale.en-US.yaml

```
@@ -0,0 +1,24 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: CologneCodeCompany.XYplorer
+PackageVersion: 26.20.0200
+PackageLocale: en-US
+Publisher: Cologne Code Company

```

```suggestion
Publisher: Donald Lessau, Cologne Code Company
```
This should match control panel

---

What control panel?